pygmt/tests/test_coast.py

```diff
@@ -37,23 +37,49 @@ def test_coast_iceland():
     return fig_ref, fig_test
 
 
-@pytest.mark.mpl_image_compare
+@check_figures_equal()
 def test_coast_aliases():
     "Test that all aliases work"
-    fig = Figure()
-    fig.coast(
-        region="-30/30/-40/40",
-        projection="m0.1i",
-        frame="afg",
-        rivers="1/1p,black",
-        borders="1/0.5p,-",
-        shorelines="0.25p,white",
-        land="moccasin",
-        water="skyblue",
-        resolution="i",
-        area_thresh=1000,
+    fig_ref, fig_test = Figure(), Figure()
+    fig_ref.coast(
+        R="-30/30/-40/40",
+        J="M25c",
+        B="afg",
+        I="1/1p,black",
+        N="1/0.5p,-",
+        W="0.25p,white",
+        G="moccasin",
+        S="skyblue",
+        D="i",
+        A=1000,
+        L="jMM+c1+w1000k+f+l",
```

Comment:
`MM` is not a valid anchor. Do you mean `CM`?

Reply:
@seisman I did mean `CM`, and changed it in my most recent commit. I don't know enough about how GMT handles a different input, but it looks like `MM` is a valid anchor; both argument variations look the same on the figure. I ran a `check_figures_equal` test for the two and it passed.

Reply:
I guess that GMT may default to "CM" if the given anchor can not be recognized.